porting/first-steps.rst

```diff
@@ -66,7 +66,7 @@ Install the required dependencies::
       zip bzr curl libc6-dev libncurses5-dev:i386 x11proto-core-dev \
       libx11-dev:i386 libreadline6-dev:i386 libgl1-mesa-glx:i386 \
       libgl1-mesa-dev g++-multilib mingw-w64-i686-dev tofrodos \
-      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
+      python3-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
```

Comment:
mhm. really? should we change it for 16.04/18.04 as well? 

I don't know. Just asking. UBports doesn't change it for 16/18: https://docs.ubports.com/en/latest/porting/introduction/Setting_up.html

Reply:
Yes,I see i'm on ubuntu 21 so relevant packages apply.

Reply:
OK, so then I assume you will update the Pr!?